src/eventer/eventer_jobq.c

```diff
@@ -209,9 +204,15 @@ eventer_jobq_create_internal(const char *queue_name, eventer_jobq_memory_safety_
     stats_rob_i32(jobq_ns, "backlog", (void *)&jobq->backlog);
     stats_rob_i64(jobq_ns, "timeouts", (void *)&jobq->timeouts);
   }
- out:
   pthread_mutex_unlock(&all_queues_lock);
   return jobq;
+
+ error_out:
+  free((void *)jobq->queue_name);
+  free((void *)jobq);
+  pthread_mutex_unlock(&all_queues_lock);
```

Comment:
dunno.. it's a potential leak that never really happens.  So the chances are low we ever even hit this label